simulator/risc_v/t/unit_test.cpp

```diff
@@ -275,3 +275,22 @@ TEST_CASE( "RISV RV64 pack")
     instr.execute();
     CHECK( instr.get_v_dst() == 0x3333'3333'2222'2222);
 }
+
+TEST_CASE( "RISV RV32 xnor")
+{
+    CHECK( RISCVInstr<uint32>(0x40e6c633).get_disasm() == "xnor $a2, $a3, $a4");
+    RISCVInstr<uint32> instr ( "xnor", 0);
+    instr.set_v_src( 0x3000'0000, 0);
+    instr.set_v_src( 0xa000'0000, 1);
+    instr.execute();
+    CHECK( instr.get_v_dst() == 0x6fff'ffff);
+}
```

Comment:
Have you run the unit tests locally?

Reply:
Yes, but with old ver.

Reply:
Please, push commits only after you are sure that unit tests pass locally. Our CI has a limited bandwidth, and there are other pull requests in progress. Running CI flow with failing traces is a waste of your and their time.

Reply:
I'm understood, thank you.